flexmeasures/data/models/user.py

```diff
@@ -102,6 +102,9 @@ def __acl__(self):
             "update": f"account:{self.id}",
         }
 
+    def get_path(self, separator: str = ">"):
```

Comment:
Does adding the unused separator here have a purpose? Is it preparing for a possible future where accounts are nested?

Reply:
The purpose was to keep the function signature unique among all the entities (`Sensor`, `Asset` and `Account`).